.github/workflows/main.yml

```diff
@@ -0,0 +1,43 @@
+name: Ruby
+on: [push,pull_request]
+jobs:
+  spec:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [ '2.6', '2.7' ]
+        rails: [ '5.0', '5.1', '5.2', '6.0', '6.1' ]
+        continue-on-error: [false]
+        include:
+          - ruby: '3.0'
+            rails: '5.2'
+            continue-on-error: true # XXX: Temporary
+          - ruby: '3.0'
+            rails: '6.0'
+            continue-on-error: false
+          - ruby: '3.0'
+            rails: '6.1'
+            continue-on-error: false
+    continue-on-error: ${{ matrix.continue-on-error }}
+    services:
+      mysql:
+        image: mysql:5.7
+        ports: ["3306:3306"]
+        env:
+          MYSQL_ROOT_USER: root
+          MYSQL_ALLOW_EMPTY_PASSWORD: 'yes'
+    name: Test on Ruby ${{ matrix.ruby }} & Rails ${{ matrix.rails }}
+    env:
+      BUNDLE_GEMFILE: gemfiles/Gemfile.${{ matrix.rails }}
```

Comment:
Syntax Highlighting may be the reason why the extension was set to .gemfile.

Reply:
Is syntax highlight for `.gemfile` major? (I haven't used...)
I don't care about that, so let's revert 6690dac